### ISSUE TITLE
Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?

### ISSUE THREAD

**Issue by feranto** [paperbits/paperbits-firebase#3]
On version 0.1.465 when trying to run the designer on paperbits-demo I'm getting the following error:

```bash
   ERROR in /paperbits-demo/node_modules/@paperbits/firebase/services/firebaseService.admin.ts
    [tsl] ERROR in /paperbits-demo/node_modules/@paperbits/firebase/services/firebaseService.admin.ts(54,13)
          TS2794: Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?
```

And it is pointing me to this fragment in firebaseService.admin.ts

```ts
    private async getFirebaseRef(): Promise<void> {
        if (this.initializationPromise) {
            return this.initializationPromise;
        }

        this.initializationPromise = new Promise(async (resolve, reject) => {
            const firebaseSettings = await this.settingsProvider.getSetting<any>("firebase");
            this.databaseRootKey = this.databaseRootKey = firebaseSettings.rootKey || "/";

            await this.applyConfiguration(firebaseSettings);

            resolve();
        });
```

Any suggestions would be appreciated...

**Comment by azaslonov** [comment]
Thanks @feranto, we'll fix this.

**Comment by azaslonov** [comment]
Fixed in version 0.1.466. Please pull the latest changes and try again.

**Comment by feranto** [comment]
Works like a charm! Thanks!
